README.md

```diff
@@ -98,12 +99,13 @@ servers == { servers, [<Server>] }
 ```
 Each server is tuple ({}):
 ```
-Server == { <SymbolicName>, { <IP>, [<Ports>] } } | { <SymbolicName>, { <IP>, [<Ports>] }, <ExtraServerOptions> }
+Server == { <SymbolicName>, { <IP>, [<Ports>] } } | { <SymbolicName>, { <IP>, [<Ports>], <ExtraSocketOptions> }, <ExtraServerOptions> }
```

Comment:
:+1: for mentioning `ExtraSocketOptions` in README.md. But could you please remove `ExtraServerOptions` from here at all. As it seems wrong documentation from some old versions of eradius. According to eradius_config.validate_server/1 (before your changes) there was only server ip and server port configurations for RADIUS servers/

Reply:
done

Reply:
thanks